test/Driver/Dependencies/bindings-build-record-options.swift

```diff
@@ -5,21 +5,21 @@
 // RUN: cd %t && %swiftc_driver -c -module-name main -driver-print-bindings ./main.swift ./other.swift ./yet-another.swift -incremental -output-file-map %t/output.json 2>&1 | %FileCheck %s -check-prefix=MUST-EXEC-INITIAL
 
 // MUST-EXEC-INITIAL-NOT: warning
-// MUST-EXEC-INITIAL: inputs: ["./main.swift"], output: {{[{].*[}]}}, condition: run-without-cascading
-// MUST-EXEC-INITIAL: inputs: ["./other.swift"], output: {{[{].*[}]}}, condition: run-without-cascading
-// MUST-EXEC-INITIAL: inputs: ["./yet-another.swift"], output: {{[{].*[}]}}, condition: run-without-cascading
+// MUST-EXEC-INITIAL: inputs: ["./main.swift", ""], output: {{[{].*[}]}}, condition: run-without-cascading
```

Comment:
These are unfortunate, the `load-module` action has empty strings for its outputs. Should these print as `""`, or should they be omitted?

Reply:
I think it's fine; it's a debugging option and we'll live with it.